azure/services/virtualnetworks/virtualnetworks_test.go

```diff
@@ -56,6 +56,25 @@ var (
 			"foo":       to.StringPtr("bar"),
 			"something": to.StringPtr("else"),
 		},
+		VirtualNetworkPropertiesFormat: &network.VirtualNetworkPropertiesFormat{
+			AddressSpace: &network.AddressSpace{
+				AddressPrefixes: &[]string{"fake-cidr"},
+			},
+			Subnets: &[]network.Subnet{
+				{
+					Name: to.StringPtr("test-subnet"),
+					SubnetPropertiesFormat: &network.SubnetPropertiesFormat{
+						AddressPrefix: to.StringPtr("fake-subnet-cidr"),
+					},
+				},
+				{
+					Name: to.StringPtr("other-test-subnet"),
+					SubnetPropertiesFormat: &network.SubnetPropertiesFormat{
+						AddressPrefix: to.StringPtr("other-fake-subnet-cidr"),
```

Comment:
Could we add a subnet with multiple CIDRs to test the `AddressPrefixes` field as well?

Reply:
added